package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "math-magician",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/react": "^13.4.0",
+    "@testing-library/user-event": "^13.5.0",
+    "big.js": "^6.2.1",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-scripts": "5.0.1",
+    "web-vitals": "^2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```

Comment:
When I run your app I get an errror.

![image](https://user-images.githubusercontent.com/38088818/190630205-6b3cec62-5328-4760-96e9-6cc66b70209e.png)

This is because your `public` and `src` directories are not in the root directory.

![image](https://user-images.githubusercontent.com/38088818/190631051-4ab276fd-8a2f-4d50-83bd-8e05e699c5db.png)

- Kindly move all the files inside the `math-magician` directory into the root directory.  The `.gitignore` and `README.md` must be in the root directory as well.

This is the proper structure:

![image](https://user-images.githubusercontent.com/38088818/190631538-7d47cd10-ac1d-48de-82e1-c4d0816ff2bb.png)